.github/workflows/cancel.yml

```diff
@@ -0,0 +1,12 @@
+name: Cancel
```

Comment:
Is this flow needed?

Reply:
not required. I will delete.

Reply:
I added this to cancel the running jobs. But not worked.